### PR TITLE
fix: Return the correct buffer for each segmentloader

### PR DESCRIPTION
**Pull request by brandonocasey**
With the current implementation a separate video and audio segment loader will try to use the combined buffered value, this causes all kinds of issues. For high bit rate sources we see a lot of append over max on source buffer errors.

### CODE REVIEW

**Comment by brandonocasey** [comment]
Added the source I used for testing, although you will have to force the 4k rendition to get it to happen.

**Comment by gkatsev** [comment]
Would be nice to add the quality-levels plugin on this page as well. I added `'node_modules/videojs-contrib-quality-levels/dist/videojs-contrib-quality-levels'` to the list of scripts in the change handler thingy. I then used `Array.from(player.qualityLevels()).forEach(l=>l.enabled = l.height > 1000)` to choose the 4k rendition

File: scripts/index.js [hunk 1]
```diff
@@ -150,6 +150,7 @@
       var urls = [
         'node_modules/video.js/dist/alt/video.core',
         'node_modules/videojs-contrib-eme/dist/videojs-contrib-eme',
+        'node_modules/videojs-contrib-quality-levels/dist/videojs-contrib-quality-levels',
```
Comment (ldayananda) [inline scripts/index.js]: this breaks the Netlify build, probably because `videojs-contrib-quality-levels` would have to be added to the devDependencies

Comment (gkatsev) [reply scripts/index.js]: Ah, whoops. Needs to be copied over in the netlify script

Comment (gkatsev) [reply scripts/index.js]: just needs to be added here https://github.com/videojs/http-streaming/blob/0d8eb578d166dbc943794bf4ecf1c3a538730e48/scripts/netlify.js#L5-L10

Comment (brandonocasey) [reply scripts/index.js]: interesting will do

Comment (brandonocasey) [reply scripts/index.js]: is there any reason that we don't just deploy out of the root project directory?

**Comment by gkatsev** [comment]
The only failing test is the live dash playback one. It is also failing unreliably, even if most of the time. When applying #545 it can reliably pass.
Going to go ahead a merge this particularly since master is frozen until all the TBA/LHLS work is in.

**Pull request merged**